app/gui2/src/components/NodeSpan.vue

```diff
@@ -73,8 +75,15 @@ watch(exprRect, (rect) => {
         :content="props.content"
         :span="child"
         :offset="childOffsets[index]!"
-        @updateExprRect="(id, rect) => emit('updateExprRect', id, rect)" /></template
-    ><template v-else>{{ exprPart }}</template></span
+        @updateExprRect="(id, rect) => emit('updateExprRect', id, rect)"
+        @updateHoveredExpr="(expr) => emit('updateHoveredExpr', expr)" /></template
```

Comment:
can be shortened to `emit('updateHoveredExpr', $event)`. not sure which we prefer though.